skabase/SKALogger/SKALogger.py

```diff
@@ -30,20 +28,14 @@
 sys.path.insert(0, basedevice_path)
 from SKABaseDevice import SKABaseDevice
 
-
-# Log related imports
-import logging
-import logging.handlers
-from logging.handlers import SysLogHandler
-
-logger_dict = {}
-logging.basicConfig()
-logger = logging.getLogger("SKALogger")
-logger.setLevel(logging.DEBUG)
-syslog = SysLogHandler(address='/var/run/rsyslog/dev/log', facility='syslog')
-formatter = logging.Formatter('%(name)s: %(levelname)s %(module)s %(message)r')
-syslog.setFormatter(formatter)
-logger.addHandler(syslog)
+# logger_dict = {}
```

Comment:
Commenting out large blocks of code isn't recommended.  Also this will likely cause a merge conflict with changes in PR #109. 

More importantly, this PR is about linting, but here is a behavioural change.

Reply:
Reverted back the changes. This is addressed in other pull request.